openfold/utils/loss.py

```diff
@@ -1033,13 +1054,16 @@ def between_residue_clash_loss(
     if asym_id is not None:
         neighbour_mask = neighbour_mask & (asym_id[..., :, None] == asym_id[..., None, :])
 
-    neighbour_mask = neighbour_mask[..., None, None]
+    neighbour_mask = neighbour_mask.detach().to('cpu').numpy()[..., None, None]
 
     c_n_bonds = (
         neighbour_mask
-        * c_one_hot[..., None, None, :, None]
-        * n_one_hot[..., None, None, None, :]
+        * c_one_hot.detach().to('cpu').numpy()[..., None, None, :, None]
```

Comment:
When I add my changes and you rebase, just make sure to remove all the cpu stuff before submitting PR

Reply:
Sure